Makefile

```diff
@@ -286,6 +286,13 @@ unit-test: ginkgo-set tidy-vendor ## Run unit tests.
 		--race --count=1 \
 		$(TEST_PKGS)
 
+.PHONY: validator-test
+validator-test: ## Run validator tests.
+	@echo "Running validator tests..."
+	@cd ./e2e/tools/validator && \
+	hatch env create && \
```

Comment:
I think it is better to have a `validator-env` target that `test` depends on

Reply:
@vprashar2929 , gentle reminder for a separate makefile for Validator :) but actually think `hatch` makes it really unnecessary to have a Makefile